src/components/Message/MessageSimple/MessageDeleted.tsx

```diff
@@ -0,0 +1,286 @@
+  groupStyle: string;
+  noBorder: boolean;
+  onLayout: (event: LayoutChangeEvent) => void;
+};
+
+type MessageDeletedPropsWithContext<
+  At extends UnknownType = DefaultAttachmentType,
+  Ch extends UnknownType = DefaultChannelType,
+  Co extends string = DefaultCommandType,
+  Ev extends UnknownType = DefaultEventType,
+  Me extends UnknownType = DefaultMessageType,
+  Re extends UnknownType = DefaultReactionType,
+  Us extends UnknownType = DefaultUserType
+> = MessageDeletedComponentProps &
+  MessageDeletedContextProps<At, Ch, Co, Ev, Me, Re, Us>;
+
+const MessageDeletedWithContext = <
+  At extends UnknownType = DefaultAttachmentType,
+  Ch extends UnknownType = DefaultChannelType,
+  Co extends string = DefaultCommandType,
+  Ev extends UnknownType = DefaultEventType,
+  Me extends UnknownType = DefaultMessageType,
+  Re extends UnknownType = DefaultReactionType,
+  Us extends UnknownType = DefaultUserType
+>(
+  props: MessageDeletedPropsWithContext<At, Ch, Co, Ev, Me, Re, Us>,
+) => {
+  const {
+    alignment,
+    formatDate,
+    getDateText,
+    groupStyle,
+    members,
+    message,
+    MessageFooter,
+    MessageStatus,
+    noBorder,
+    onLayout,
+    otherAttachments,
+    showMessageStatus,
+  } = props;
+
+  const {
+    theme: {
+      colors: {
+        grey,
+
+        grey_whisper,
+      },
+      messageSimple: {
+        content: {
+          container: { borderRadiusL, borderRadiusS },
+          deletedContainer,
+          deletedContainerInner,
+          deletedText,
+        },
+      },
+    },
+  } = useTheme();
+
+  return (
+    <View
+      onLayout={onLayout}
+      style={[
+        alignment === 'left' ? styles.leftAlignItems : styles.rightAlignItems,
+        deletedContainer,
+      ]}
+    >
+      <View
+        style={[
+          styles.containerInner,
+          {
+            backgroundColor: grey_whisper,
+            borderBottomLeftRadius:
+              groupStyle === 'left_bottom' || groupStyle === 'left_single'
+                ? borderRadiusS
+                : borderRadiusL,
+            borderBottomRightRadius:
+              groupStyle === 'right_bottom' || groupStyle === 'right_single'
+                ? borderRadiusS
+                : borderRadiusL,
+            borderColor: grey_whisper,
+          },
+          noBorder ? { borderWidth: 0 } : {},
+          deletedContainerInner,
+        ]}
+        testID='message-content-wrapper'
+      >
+        <MessageTextContainer<At, Ch, Co, Ev, Me, Re, Us>
+          markdownStyles={merge({ em: { color: grey } }, deletedText)}
+          message={{ ...message, text: '_Message deleted_' }}
+        />
+      </View>
+      <MessageFooter
+        alignment={alignment}
+        formattedDate={getDateText(formatDate)}
+        isDeleted
+        members={members}
+        message={message}
+        MessageStatus={MessageStatus}
+        otherAttachments={otherAttachments}
+        showMessageStatus={showMessageStatus}
```

Comment:
Let's take this opportunity to get rid of some of this prop drilling. You can just access these values from corresponding context in MessageFooter or MessageDeleted.